core/src/banking_stage/consume_worker.rs

```diff
@@ -572,6 +538,42 @@ pub(crate) mod external {
             Ok(())
         }
 
+        fn send_response(
```

Comment:
nit - let's call it `send_execution_response` since it's specifically for execution